src/backend/aspen/api/views/tests/test_health.py

```diff
@@ -0,0 +1,11 @@
+from fastapi.testclient import TestClient
+
+from aspen.api.main import app
+
+client = TestClient(app)
```

Comment:
I'm unclear on when we want to use `TestClient` from FastAPI vs the fixture `http_client` (which looks like a "real" client) -- is it that the first is used for synchronous stuff, while the second is for async stuff? Or is it for when we need to test DB interactions?